server/database.ts

```diff
@@ -0,0 +1,44 @@
+// import sqlite from bun
+import { Database } from "bun:sqlite";
+
+// user table
+export interface User {
+    Username: string;
+    Password: string;
+    Salt: string;
+    ProfilePicture: Blob;
+}
+
+// post table
+export interface Post {
+    Username: string;
+    Content: string;
+    Picture: Blob;
+    Timestamp: number;
+}
+
+// typeguard for user
+export function isUser(x: any): x is User {
+    return "Username" in x && "Password" in x && "Salt" in x && "ProfilePicture" in x;
+}
+
+// typeguard for post
+export function isPost(x: any): x is Post {
+    return "Username" in x && "Content" in x && "Picture" in x && "Timestamp" in x;
+}
+
+// EXAMPLE IMPORT:
+// import { getDB } from "./database.ts"
+// const db = getDB();
+
+export function getDB(): Database {
```

Comment:
Just as a nice testing feature, we should be able to specify the location of the file (i.e. what is passed into the Database constructor). Also a rename to `initDB` would be nice for clarity.